packages/vincent-tools/vincent-tool-uniswap-swap/CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to Vincent Tool Uniswap Swap
+
+This document provides guidelines for contributing to the Vincent Tool Uniswap Swap project.
+
+## Overview
+
+The Vincent Tool Uniswap Swap is a tool to trigger swaps on Uniswap from a Vincent app on behalf of the delegator. It's part of the Vincent Tools ecosystem and is built using the Vincent Tool SDK.
+
+## Setup
+
+1. Follow the global setup instructions in the repository root [CONTRIBUTING.md](../../../CONTRIBUTING.md).
+2. Install dependencies:
+   ```bash
+   pnpm install
+   ```
+
+## Development Workflow
+
+### Building
+
+Build the tool:
+```bash
+nx build vincent-tool-uniswap-swap
+```
+
+### Testing
+
+Run tests:
+```bash
+nx test vincent-tool-uniswap-swap
+```
+
+### Deployment
+
+Deploy the tool:
+```bash
+nx deploy vincent-tool-uniswap-swap
```

Comment:
See other comment around `action:deploy`

Reply:
Changed it in all LAs